app/graphql/mutations/users/mutation_manifest.rb

```diff
@@ -0,0 +1,10 @@
+module Mutations
+  module Users
+    module MutationManifest
+      extend ActiveSupport::Concern
+      included do
+        field :update_user, resolver: Mutations::Users::UpdateUserDetailsMutation
```

Comment:
single mutation to update all the field on the core user, *note* not the GithubUser model

Reply:
👍